packages/fxa-settings/README.md

```diff
@@ -9,6 +9,77 @@ This documentation is up to date as of 2020-08-03.
 - `yarn test` to run unit tests
 - `yarn storybook` to open Storybook
 
+### GQL and REST API Calls
+
+This package consumes the [FxA GraphQL API](https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api) for the large majority of communication with the server. See the documentation to connect to the playground to view the API docs and schema.
+
+The application sends queries to the GQL server to request and receive data, and sends mutations for all other request types with aid of Apollo Client and Apollo Provider. See the [GQL documentation](https://graphql.org/learn/) to learn more.
```

Comment:
This took a few reads before the sentence made sense to me. Just a pitch, but how about this:

> FxA Settings uses the GQL server, with the help of [Apollo](https://www.apollographql.com/docs/) Client and Provider, to interact with our Auth server. Requests that retrieve data are called queries, and requests that modify or delete data are called mutations. See the [GQL documentation](https://graphql.org/learn/) to learn more.

Reply:
Lol, this is what I get for hastily throwing up a draft, I just reread this and confused myself. I'll take what you suggested and add a couple tweaks. I'll go ahead and convert this to a regular ol PR with my next push.